src/components/design-system.css

```diff
@@ -268,11 +268,12 @@ a:hover {
     transition: transform ease-in-out 0.7s, opacity ease-in-out 0.7s;
     opacity: 0;
     pointer-events: none;
+    min-width: 135px;
   }
   .isScrolled .get-an-estimate.show-when-scrolled {
     opacity: 1;
     pointer-events: all;
-    transform: translateX(-15%);
+    transform: translateX(-5%);
```

Comment:
I also think @tkarssli and @iamvictorli might have some more context on UniversalNavbar if y'all can drop in some thoughts on this PR? Is this transform and min width going to break anything that you can forsee?

Reply:
For context here, the change to this transform is because there is less space available on mobile with the addition of the search icon. I adjusted this to reduce overlap of the animated CTA with other navbar elements.

Reply:
I'll request reviews when this is all ready to go @tkarssli @iamvictorli, if you want to save your code review until then.